### DISCUSSION TITLE
Change mount sshfs options -- removing transform_smylinks

### DISCUSSION THREAD

**Discussion by marcnarc** [canonical/multipass#3795] [Q&A] [answered]
Hi!

I would like to mount a directory from my host into a guest _without_ the `-o transform_smylinks` option. Is this possible?

I'm using Ubuntu Desktop noble 24.04 with `multipass` 1.14.1 with the `libvirt` backend.

In my guest the `sshfs` command is given the unwanted option:
```
5021 sudo env LD_LIBRARY_PATH=/snap/multipass-sshfs/145/lib /snap/multipass-sshfs/145/bin/sshfs -o slave -o transform_symlinks -o allow_other -o Compression=no -o dir_cache=no :/home/marcnarc/Code /home/ubuntu/Code/.
5022 /snap/multipass-sshfs/145/bin/sshfs -o slave -o transform_symlinks -o allow_other -o Compression=no -o dir_cache=no :/home/marcnarc/Code /home/ubuntu/Code/.
```

A "native" mount might work around this, but that seems to be unsupported with the `libvirt` backend:
```
> multipass mount --type=native Code guest:Code
mount failed: The native mounts feature is not implemented on this backend.
```

(I suspect this option comes from Ubuntu's snap confinement "protection", so I apologize if I'm off-topic here. I would greatly appreciate being pointed at the right forum.)

**Comment by andrei-toterman** [comment, marked as answer]
Hey, @marcnarc! Using just the capabilities provided by Multipass, it is not possible to customize the parameters passed to sshfs. That options was deemed to be appropriate for most use-cases, so unfortunately, if you want to remove it, you'd need to manually manage your own sshfs process. Sorry!
